examples/advanced/tidb-monitor.yaml

```diff
@@ -375,10 +375,10 @@ spec:
     ## Base image of the initializer.
     baseImage: pingcap/tidb-monitor-initializer
     ## ImagePullPolicy of initializer.
-    imagePullPolicy: IfNotPresent
+    # imagePullPolicy: IfNotPresent
     ## Initializer version.
-    version: v5.2.0
-    
+    version: v5.2.1
```

Comment:
How about updating this version until the next release with our shell command?

Reply:
Yes, this is the version that is missed in the last update. Now all the versions are v5.2.1 and we can update to v5.3.0 in the next release.